### ISSUE TITLE
Android: Sensor returns correct orientation but `onOrientationChanged(useSensor: true)` does not fire.

### ISSUE THREAD

**Issue by MRobertEvers** [rmtmckenzie/flutter_native_device_orientation#32]
It appears `onOrientationChanged(useSensor: true)` does not fire an event when the sensor orientation changes. However, the correct orientation is reported when the sensor is read manually.

This can be seen in android on the official example.

Any suggested workarounds? I may get around to fixing this myself, but I don't have time right now.

**Comment by MRobertEvers** [comment]
Additional info, `NativeDeviceOrientationReader` widget works.

```
NativeDeviceOrientationReader(
  builder: (context) { },
  useSensor: true,
)
```

**Comment by rmtmckenzie** [comment]
I haven't been able to reproduce this. Closing unless I receive additional feedback.

**Comment by LeNgocHiep** [comment]
same issue on version 1.1.4, working on Samsung A53 but not working on Samsung S10+. NativeDeviceOrientationReader widget works.

**Comment by rmtmckenzie** [comment]
Can you give me a bit more information? Does it never call back at all on Samsung S10+ no matter which way you turn the device? Do you have auto-rotate enabled? Is the activity rotating or set to not rotate?

**Comment by LeNgocHiep** [comment]
Yes, never call back at all on Samsung S10+. This package works for me https://pub.dev/packages/device_orientation. I have some config below

![image](https://github.com/rmtmckenzie/flutter_native_device_orientation/assets/28060833/ceae7b4a-d1cb-4ec5-b1d1-2754026fe1d9)
![image](https://github.com/rmtmckenzie/flutter_native_device_orientation/assets/28060833/63933b15-e9a9-4282-8bb6-82cbd88137c2)

**Comment by desmeit** [comment]
same issue here, on Pixel 6a `useSensor` is not working.
